index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Dani's FullStack Web Developer Portfolio</title>
+</head>
+<body>
+  <header class="app-bar">
+    <div class="status-bar"></div>
+    <div class="toolbar">
+      <div class="log-placeholder">
+        Welcome
+      </div>
+      <div class="hamburger-icon">
+        <img src="images/icons/hamburger-icon.svg" alt="hamburger-icon">
+      </div>
```

Comment:
Kindly wrap this element in a `<nav>` tag instead of a `<div>`🙏. This is for semantic reasons as the hamburger button links to the mobile menu and is hence a navigation element